cassandra/src/main/java/org/kontext/cassandra/documents/DocumentRepositoryImpl.java

```diff
@@ -40,14 +50,15 @@ private void init() {
 		ensureDocumentTableExistence();
 
 		String cqlMask = "INSERT INTO %s (id, html, raw_text, link_count, create_date) "
-				+ "VALUES (?, ?, ?, ?, toTimestamp(now()));";
+				+ "VALUES (?, ?, ?, ?, toTimestamp('" + todaysDate + "'));";
```

Comment:
This is to reduce the uniqueness of time stamp. Making the full date time format to just date makes partitioning simpler.